.gitattributes

```diff
@@ -1,25 +1,25 @@
 * text=auto eol=lf
```

Comment:
How are these ordered now? It's adding a comment

Reply:
each type of files gets sorted alphabetically, i.e all generated files alphabetically then all special syntax files alphabetically (special syntax being when the language of a file is specified). should i add a comment?